src/javascript/events/component-view.js

```diff
@@ -80,6 +79,6 @@ const init = (opts = {}) => {
 	elementsToTrack.forEach(el => observer.observe(el));
 };
 
-export default init;
+export default { init };
```

Comment:
I guess this would constitute a breaking change? If so, maybe it would be better to update the documentation/migration-guide to mention that it no exports an object?

Reply:
the change to this *was* an undocumented breaking change, it has broken `next-search-page`. i think this constitutes a patch

Reply:
the migration guide says that what has changed between versions 1 and 2 is limited to the main interface https://github.com/Financial-Times/o-tracking/blob/master/MIGRATION.md

but that isn't true, because this change also took place and breaks next-search-page.

`modules.exports` was an object containing `init` before: https://github.com/Financial-Times/o-tracking/blob/v1.7.3/src/javascript/events/component-view.js#L85-L87

so when you accessed `oTracking.view` you would have an object containing a function called `init`, but now `oTracking.view` is a function

Reply:
the README also contains several mentions to `oTracking.view.init()` which is no longer possible:
https://github.com/Financial-Times/o-tracking/tree/v1.7.3#parameters

Reply:
you can see that we changed the tests to fit with the new api without documenting the api had been changed: https://github.com/Financial-Times/o-tracking/commit/a2de177e2dcc82edbcf94409c8357e9ba3a2287c#diff-3678da481657f896f0654a97774b6694L87-L88